gooddata-pandas/gooddata_pandas/data_access.py

```diff
@@ -446,7 +446,10 @@ def compute_and_extract(
     if not exec_def.has_attributes():
         return _extract_for_metrics_only(response, cols, col_to_metric_idx), dict()
     else:
-        attributes = sdk.catalog_workspace_content.get_attributes_catalog(workspace_id, include=["labels", "datasets"])
+        filter_query = filter_for_attributes_labels(exec_def.attributes)
+        attributes = sdk.catalog_workspace_content.get_attributes_catalog(
+            workspace_id, include=["labels", "datasets"], rsql_filter=filter_query
```

Comment:
Are we sure this will all fit into URL not hitting the length limit?

Reply:
It is a good point. I tried it, and it really can hit the limitation.

```
gooddata_api_client.exceptions.ApiException: Status Code: 414
Reason: Request-URI Too Large
HTTP response headers: HTTPHeaderDict({'Server': 'nginx', 'Date': 'Fri, 17 Jan 2025 13:07:20 GMT', 'Content-Type': 'text/html; charset=UTF-8', 'Connection': 'close'})
HTTP response body: <html>
<head><title>414 Request-URI Too Large</title></head>
<body>
<center><h1>414 Request-URI Too Large</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

Reply:
The limit for URLs is 2048 characters. I will limit rsql filter to 1500 characters. If it is higher, it will fetch all attributes.